src/components/atoms/UiLink/UiLink.vue

```diff
@@ -59,13 +59,14 @@ const {
   @include mixins.inner-border($element, $color: transparent, $width: 0, $radius: var(--border-radius-button));
   @include mixins.font($element, body-1);
 
-  display: inline;
+  display: flex;
```

Comment:
```suggestion
  display: inline;
```
it should be `inline` because it's allow to use link in text without many problems...
`display:flex`
![Zrzut ekranu 2022-12-8 o 13 04 13](https://user-images.githubusercontent.com/12138170/206444470-e17f93b3-f1f8-4ff3-b1c2-fc0ec5b0bbac.png)
`display:inline-flex`
![Zrzut ekranu 2022-12-8 o 13 03 58](https://user-images.githubusercontent.com/12138170/206444535-3464ce48-9e43-483b-a17e-dafb1b2aeea6.png)
`display:inline`
![Zrzut ekranu 2022-12-8 o 13 04 38](https://user-images.githubusercontent.com/12138170/206444566-d6f64065-7b7f-4532-bc53-0280c13d38a5.png)
it's only one way to keep link in text behaviour